projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-extra-details/user-content/user-contributions/user-build-card/user-build-card.component.scss

```diff
@@ -1,4 +1,4 @@
-@import "projects/commudle-admin/src/assets/styles/variables";
+@import "../../../../../../../../../assets/styles/variables";
```

Comment:
This should be an absolute path like the red line above not a relative one...

Reply:
Done!